tests/components/vera/test_init.py

```diff
@@ -160,19 +218,18 @@ async def test_exclude_and_light_ids(
     component_data = await vera_component_factory.configure_component(
         hass=hass,
         controller_config=new_simple_controller_config(
+            config_source=ConfigSource.CONFIG_ENTRY,
             devices=(vera_device1, vera_device2, vera_device3, vera_device4),
             config={**{CONF_CONTROLLER: "http://127.0.0.1:123"}, **options},
         ),
     )
 
     # Assert the entries were setup correctly.
     config_entry = next(iter(hass.config_entries.async_entries(DOMAIN)))
-    assert config_entry.options == {
-        CONF_LIGHTS: [4, 10, 12],
-        CONF_EXCLUDE: [1],
-    }
+    assert config_entry.options[CONF_LIGHTS] == [4, 10, 12]
+    assert config_entry.options[CONF_EXCLUDE] == [1]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)